docs/installation/raspbian.md

```diff
@@ -50,92 +53,87 @@ sudo python3 get-pip.py
 
 ## Raspberry Pi configuration
 
-This section deals with the special configuration needed to get kalliope working on a RPi.
+This section deals with the special configuration needed to get Kalliope working on a RPi.
 
-### Microphone configuration
+### Pulseaudio configuration
 
-Get the output card:
+Add the pi user to the group pulse-access to allow to use the audio device:
+
+```bash
+sudo usermod -a -G pulse-access pi
+``` 
+
+Setting up default client settings by adding default server and disable autospawn to client.conf:
 ```bash
-aplay -l
+printf '\ndefault-server = /var/run/pulse/native\nautospawn = no' | sudo tee -a /etc/pulse/client.conf
 ```
 
-Output example with a USB headset connected:
+Get the pulseaudio service script and move it to /etc/systemd/system/pulseaudio.service:
 ```bash
-**** List of PLAYBACK Hardware Devices ****
-card 0: ALSA [bcm2835 ALSA], device 0: bcm2835 ALSA [bcm2835 ALSA]
-  Subdevices: 7/8
-  Subdevice #0: subdevice #0
-  Subdevice #1: subdevice #1
-  Subdevice #2: subdevice #2
-  Subdevice #3: subdevice #3
-  Subdevice #4: subdevice #4
-  Subdevice #5: subdevice #5
-  Subdevice #6: subdevice #6
-  Subdevice #7: subdevice #7
-card 0: ALSA [bcm2835 ALSA], device 1: bcm2835 ALSA [bcm2835 IEC958/HDMI]
-  Subdevices: 1/1
-  Subdevice #0: subdevice #0
-card 1: Headset [Logitech USB Headset], device 0: USB Audio [USB Audio]
-  Subdevices: 0/1
-  Subdevice #0: subdevice #0
+wget https://raw.githubusercontent.com/kalliope-project/kalliope/master/install/files/pulseaudio.service && sudo mv pulseaudio.service /etc/systemd/system/
```

Comment:
I would prefer here to show the content of the service instead of downloading a file. Like it is done for the kalliope service.

Reply:
I was going to make it as simple as possible for the user, this way they just need to copy and paste, cause this is necessary step for the installation, other then the kalliope service. 

What about additionally adding the content of the service to the docs, so the user can see the content and choose to download directly and move or to create the file himself?

Reply:
Ok for both then.